src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -2780,4 +2783,17 @@ public void sendMessage(@NotNull ChatMessageType position, @NotNull BaseComponen
 
 	}
 
+	@Override
+	public void setFlyingFallDamage(@NotNull TriState flyingFallDamage)
+	{
+		Preconditions.checkNotNull(flyingFallDamage);
+		this.flyingFallDamage = flyingFallDamage;
+	}
+
+	@Override
+	public @NotNull TriState hasFlyingFallDamage()
+	{
+		return this.flyingFallDamage;
+	}
```

Comment:
These should be implemented in their own PR

Reply:
Thought so